core/src/main/resources/hudson/PluginManager/advanced.jelly

```diff
@@ -28,16 +28,16 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-    <l:layout title="${%Advanced Settings} - ${%Plugin Manager}" permission="${app.SYSTEM_READ}">
+    <l:layout title="${%Advanced settings} - ${%Plugin Manager}" permission="${app.SYSTEM_READ}">
```

Comment:
Is `Detail - Category` a reasonable presentation universal across languages Jenkins is translated to?

Reply:
That's a good question and something I haven't got a concrete answer to. I've given it a quick Google but haven't found any articles on the topic, so I've looked at how various sites behave with right-to-left languages:

Google Search - Doesn't change the hierarchy of the title (e.g. Jenkins - Google)
Wikipedia - Does change the hierarchy of the title (e.g. Wikipedia - Jenkins)
Google Translate - Does change the hierarchy of the title